docs/docusaurus/docs/guides/expectations/creating_custom_expectations/add_custom_parameters.md

```diff
@@ -0,0 +1,91 @@
+   parse_strings_as_datetimes: bool = False,
+   allow_cross_type_comparisons=None,       
+   **kwargs   
+):
+```
+To view the full code that you would use to pass custom parameters in a Custom Expectation, see [expect_column_values_to_be_lat_lon_coordinates_in_range_of_given_point.py](https://github.com/great-expectations/great_expectations/blob/develop/contrib/great_expectations_geospatial_expectations/great_expectations_geospatial_expectations/expectations/expect_column_values_to_be_lat_lon_coordinates_in_range_of_given_point.py).
+
+## kwarg-based access
+
+GX recommends using custom parameters to create validations and optimize workflows. However, you can also use `kwarg` syntax to define Metrics for your Expectations. An example `kwarg` implementation is provided in [expect_column_values_to_be_lat_lon_coordinates_in_range_of_given_point.py](https://github.com/great-expectations/great_expectations/blob/develop/contrib/great_expectations_geospatial_expectations/great_expectations_geospatial_expectations/expectations/expect_column_values_to_be_lat_lon_coordinates_in_range_of_given_point.py).
+
+## Allowable parameters
+
+The following table lists the allowable parameters that are available for each Expectation class.
+
+| Expectation class                 | Allowable parameters           |
+| ----------------------------------| -------------------------------|
+| BatchExpectation                  | See [BatchExpectation](https://docs.greatexpectations.io/docs/reference/api/expectations/expectation/BatchExpectation_class)             |
+| ColumnAggregateExpectation        | See [ColumnAggregateExpectation](https://docs.greatexpectations.io/docs/reference/api/expectations/expectation/ColumnAggregateExpectation_class)              |
+| ColumnMapExpectation              | See [ColumnMapExpectation](https://docs.greatexpectations.io/docs/reference/api/expectations/expectation/ColumnMapExpectation_class)              |
+| RegexBasedColumnMapExpectation    | See [RegexBasedColumnMapExpectation](https://docs.greatexpectations.io/docs/reference/api/expectations/regex_based_column_map_expectation/RegexBasedColumnMapExpectation_class)              |
+| SetBasedColumnMapExpectation      | See [SetBasedColumnMapExpectation](https://docs.greatexpectations.io/docs/reference/api/expectations/set_based_column_map_expectation/SetBasedColumnMapExpectation_class)              |
+| ColumnPairMapExpectation          | See [ColumnPairMapExpectation](https://docs.greatexpectations.io/docs/reference/api/expectations/expectation/ColumnPairMapExpectation_class)              |
+| MulticolumnMapExpectation         | See [MulticolumnMapExpectation](https://docs.greatexpectations.io/docs/reference/api/expectations/expectation/MulticolumnMapExpectation_class)              |
+| QueryExpectation                  |  See [QueryExpectation](https://docs.greatexpectations.io/docs/reference/api/expectations/expectation/QueryExpectation_class)              |
```

Comment:
Technically *any* parameters are allowed, as they're custom and created by the users. Are these intended to be the default parameters available?

Reply:
I'm thinking this may be better phrased as "default parameters" or "available default parameters" rather than "allowable parameters"?